CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/lib/Event.kt

```diff
@@ -0,0 +1,22 @@
+package co.nimblehq.coroutine.lib
+
+/**
+ * Used as a wrapper for data that is exposed via a LiveData that represents a single event.
+ */
+open class Event<out T>(val content: T) {
```

Comment:
why do we need to keep this class as open?

Reply:
No need @luongvo I will remove it, thanks for reminding me 🙏